server/channels/app/webhook.go

```diff
@@ -734,12 +735,12 @@ func (a *App) HandleIncomingWebhook(c request.CTX, hookID string, req *model.Inc
 	}()
 
 	if req == nil {
-		return model.NewAppError("HandleIncomingWebhook", "web.incoming_webhook.parse.app_error", nil, "", http.StatusBadRequest)
+		return model.NewAppError("HandleIncomingWebhook", "web.incoming_webhook.parse.app_error", map[string]any{"hook_id": hookID}, "", http.StatusBadRequest)
```

Comment:
This only apply if we keep the hook id in the error.

The same way as with `Logger().With()`, I wonder if it is reasonable to define a variable for this (e.g. `errorDefaultProps`, or any other better naming), so we don't have to write every time `map[string]any{"hook_id": hookID}`.

What do you think?